packages/atomic-layout/package.json

```diff
@@ -28,6 +28,7 @@
     "bundlesize:esm": "bundlesize -f lib/esm/index.js",
     "cypress": "cypress open --env envName=dev",
     "cypress:cli": "cypress run --spec=./examples/all.test.js --browser=chrome --env envName=ci",
+    "jest": "jest",
```

Comment:
what is this for?

Reply:
To run the `jest` command from the `packages/atomic-layout` directory. Otherwise you'd have to run `../../node_modules/.bin/jest`. I don't have `jest` installed globally, and generally don't recommend it.